languages/csharp-dotnet-core2/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "artifacts": [
+        {"source":"/opt/algorithm/bin/Release/*/*.dll", "destination":"/opt/algorithm/Algorithm.dll"},
+        {"source":"/opt/algorithm/bin/Release/*/*.runtimeconfig.json", "destination": "/opt/algorithm/Algorithm.runtimeconfig.json"},
+        {"source":"/opt/algorithm/resources/", "destination": "/opt/algorithm/resources/"},
```

Comment:
bad json. get rid of comma at the end of this line

Reply:
whoops, fixed.